bert_e/git_host/github/__init__.py

```diff
@@ -43,14 +42,14 @@ def __init__(self, login: str, password: str, email: str, org=None,
             'Accept': 'application/vnd.github.v3+json',
             'User-Agent': 'Bert-E',
             'Content-Type': 'application/json',
-            'From': email
+            'From': email,
+            'Authorization': 'token ' + password
```

Comment:
We are concatenating password and token ?

Reply:
yep, we add the content of password to the string "token "